build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/precommit/transport/TransportTestExistPrecommitPlugin.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.gradle.internal.precommit.transport;
+
+import org.elasticsearch.gradle.internal.conventions.precommit.PrecommitPlugin;
+import org.elasticsearch.gradle.util.GradleUtils;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.dsl.DependencyHandler;
+import org.gradle.api.file.FileCollection;
+import org.gradle.api.plugins.JavaPlugin;
+import org.gradle.api.tasks.SourceSet;
+import org.gradle.api.tasks.TaskProvider;
+
+import java.util.Map;
+
+public class TransportTestExistPrecommitPlugin extends PrecommitPlugin {
+    @Override
+    public TaskProvider<? extends Task> createTask(Project project) {
+
+        try {
```

Comment:
Why having that try catch block?

Reply:
fair point - there is no special handling for exception. will remove